director/director_api.go

```diff
@@ -33,3 +49,78 @@ func ListNamespacesFromOrigins() []NamespaceAd {
 	}
 	return namespaces
 }
+
+func LoadDirectorPublicKey() (*jwk.Key, error) {
+	directorUrlStr := param.Federation_DirectorUrl.GetString()
+	if len(directorUrlStr) == 0 {
+		return nil, errors.Errorf("Director URL is unset; Can't load director's public key")
+	}
+	log.Debugln("Director URL:", directorUrlStr)
+	directorUrl, err := url.Parse(directorUrlStr)
+	if err != nil {
+		return nil, errors.Wrap(err, fmt.Sprintln("Invalid director URL:", directorUrlStr))
+	}
+	directorUrl.Scheme = "https"
+
+	discoveryUrl, _ := url.Parse(directorUrl.String())
```

Comment:
Elsewhere in Pelican we use `discoveryUrl` to indicate the Federation's discovery url. Can you modify the variable name to make it clear this is meant to be the Director's discovery url?